push-files/push.test.js

```diff
@@ -1,19 +1,22 @@
 process.env.GITHUB_RUN_NUMBER = '2345';
 
-const { buildMetadata } = require('./push');
-
 jest.mock('@actions/exec', () => ({
-  exec: jest.fn(),
+  exec: jest.fn((cmd, args, options) => {
+    options.listeners.stdout('638b46454a5u95520e07');
+  }),
```

Comment:
Not really sure why ESLint is failing here. Locally it's passing and from my understanding this format is valid / expected. @ahume any idea why this behaves different on CI than local?

Reply:
Yeah, that's weird. There's two errors on the same line. One that wants a comma and one that doesn't!

Reply:
Hm, the current checks aren't failing though. Are these from old runs?

Reply:
No, I pushed another change this morning, although I reverted that change later with a force push. In general I have the feeling force pushes doesn't work well with GitHub Actions.